packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,47 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) file system to a local directory. 
+
+The following sample enables the lambda function to mount the filesystem from Amazon EFS via the `accessPoint` to 
+the local `/mnt/msg` directory.
+
+```ts
+const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
+  fileSystem,
+  createAcl: {
+    ownerGid: '1000',
+    ownerUid: '1000',
+    permissions: '755',
+  },
+  path: '/lambda',
+  posixUser: {
+    uid: '1000',
+    gid: '1000',
+  },
+});
+
+const fn = new lambda.Function(stack, 'MyLambda', {
+  code,
+  handler,
+  runtime,
+  vpc,
+  vpcSubnets: {
+    subnetType: ec2.SubnetType.PRIVATE,
+  },
+  securityGroups: fileSystem.connections.securityGroups,
```

Comment:
Is this required? If it is, I expect it to be implicit and automatic. If it is not required, omit from the snippet.

Reply:
`vpcSubnets` should not be required, but I am afraid `securityGroups` is.

To allow lambda function to access the efs filesystem, the lambda function will need a security group  to connect to efs  via TCP on the NFS port 2049. The general practice should be add the `fileSystem.connections.securityGroups` into the Lambda function security groups and allow the fileSystem.connections connect internally on its default port. However, as lambda.Function can't access the `fileSystem`, we are not able to add them for users.  So we probably have to leave this for users under the hood. Any recommended approaches?